### PR TITLE
Functest for issue 2248, issue fixed when 2246 was fixed (release/1.2.0)

### LINKED ISSUE #2248: Forwarding Updates overwrites the types of attributes
When doing a `PUT /v2/entities/<eid>/attrs/<attrName>/value`, no attribute type is given and the default value of the attribute - `none` - is used in the forwarded request, and this makes the remote (in context provider) attribute change type to `none` - overwriting its correct type.

Must be tested both with JSON values and other types (String, Number, Bool, etc)

### PR DESCRIPTION
**Pull request by kzangeli**
#2248 already fixed - this is just a functest to make sure it 'stays fixed'

### CODE REVIEW

File: test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Create E1/T1/A1 in CP1, A1 with type 'string'
+# 02. Register E1/T1/A1 in CB, as belonging to CP1
+# 03. Update E1/T1/A1 in CB, using PUT /v2/entities/E1/attrs/A1/value, without stating type of the attribute
+# 04. Make sure that the value of E1/T1/A1 is updated in CP1, but that its attribute type stays intact (as 'string')
+#
```
Comment (fgalan) [inline test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test]: Missing steps 5 and 6 in the preamble. In addition, the wording for 3 and 5 should tell which case is tested in each case (i.e. one based in application/json and the other based in text/plain).

Comment (kzangeli) [inline test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test]: Ah yes, I forgot about that

Comment (kzangeli) [inline test/functionalTest/cases/2248_forwarding_updates_overwrite_the_types_of_attributes/forwarding_updates_overwrite_the_types_of_attributes.test]: Fixed in 3ede5f1

**Comment by fgalan** [comment]
LGTM

Branch not deleted, to reuse for twin PR on develop

**Pull request merged**